.gitignore

```diff
@@ -72,3 +72,10 @@ scDebugLog
 coverage.json
 coverage/
 coverageEnv/
+
+node_modules
```

Comment:
I also see that package-lock.json is removed. Will we still get reproducible builds?